bpf/lib/common.h

```diff
@@ -247,6 +261,8 @@ enum {
 #define DROP_PROXYMAP_CREATE_FAILED	-161
 #define DROP_POLICY_CIDR		-162
 
+#define DIRECTION_INGRESS 0
```

Comment:
I think an enum would be better then #define each direction.

Reply:
sure.

Reply:
done